cdc/processor/processor.go

```diff
@@ -365,7 +365,11 @@ func (p *processor) handleTableOperation(ctx cdcContext.Context) error {
 				if opt.BoundaryTs < globalCheckpointTs {
 					log.Warn("the BoundaryTs of remove table operation is smaller than global checkpoint ts", zap.Uint64("globalCheckpointTs", globalCheckpointTs), zap.Any("operation", opt))
 				}
-				table.AsyncStop(opt.BoundaryTs)
+				if !table.AsyncStop(opt.BoundaryTs) {
```

Comment:
If table `AsyncStop` returns false here, will it be called later?

Reply:
Yes. `EtcdWorker` will tick regularly and the processor will attempt to handle unfinished operations each tick.